modules/cluster-logging-updating-logging.adoc

```diff
@@ -264,7 +264,7 @@ cronjob.batch/elasticsearch-rollover-infra
 +
 You should see the `elasticsearch-delete-\*` and `elasticsearch-rollover-*` indices.
 
-== Post-update tasks
+.Post-update tasks
 
-If you use Kibana, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.6, you must recreate your Kibana index patterns and visualizations.
+If you use the Log Forwarding API to forward logs, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.6, you must update your Log Forwarding custom resource to use some renamed parameters.
```

Comment:
...replace the LogForwarding custom resource with an instance of ClusterLogForwarder